localdev/config/armada/base.yaml

```diff
@@ -0,0 +1,29 @@
+corsAllowedOrigins: ["http://localhost:3000", "http://localhost:8089"]
+auth:
+  basicAuth:
+    enableAuthentication: false
+  anonymousAuth: true
+  permissionGroupMapping:
+    submit_jobs: ["everyone"]
+    submit_any_jobs: ["everyone"]
+    create_queue: ["everyone"]
+    delete_queue: ["everyone"]
+    cancel_jobs: ["everyone"]
+    cancel_any_jobs: ["everyone"]
+    reprioritize_jobs: ["everyone"]
+    reprioritize_any_jobs: ["everyone"]
+    watch_events: ["everyone"]
+    watch_all_events: ["everyone"]
+    execute_jobs: ["everyone"]
+pulsar:
```

Comment:
I'm unsure what this option does since you also add pulsar.  I think we should remove this as its unclear to me if we actually enabled pulsar.

Reply:
okay will remvoe